app/presenters/dashboard_presenter.rb

```diff
@@ -87,18 +104,18 @@ def users
   def bar_chart_value(user:)
     return if user.time_ranges.none?
 
-    UserStatsPresenter.new(
-      user: user,
-      actual_id: @params[:actual_id],
-      filter_start_date: filter_start_date,
-      filter_end_date: filter_end_date
-    ).percentage_delivered
+    user_presenter(user).percentage_delivered
```

Comment:
user_presenter or it's instance of UserStatsPresenter ? if so variable name should be user_stats_presenter for consistency